### PR TITLE
Dev: Hide lsb and service resource agent class type from UI and doc

### PR DESCRIPTION
**Pull request by liangxin1300**

### CODE REVIEW

File: crmsh/ra.py [hunk 1]
```diff
@@ -50,8 +50,9 @@ def ra_classes():
         return cache.retrieve("ra_classes")
     if can_use_crm_resource():
         l = crm_resource("--list-standards")
+        l = [x for x in l if x not in ("lsb", "service")]
     else:
-        l = ["heartbeat", "lsb", "ocf", "stonith", "systemd"]
```
Comment (nicholasyang2022) [inline crmsh/ra.py:54]: How is `heartbeat` related to `lsb`?

**Review by nicholasyang2022 — COMMENTED** [review]
Is there a PED to be referenced?

**Comment by zzhou1** [comment]
> Is there a PED to be referenced?

Let's wait few moment till the Impl. Task PED get created.

File: crmsh/ra.py (hunk 1 above)
Comment (zzhou1) [reply crmsh/ra.py:54]: "heartbeat" is used by the current ocf1.0 and 1.1 standard and located at
/usr/lib/ocf/resource.d/heartbeat

**Comment by liangxin1300** [comment]
> > Is there a PED to be referenced?
> 
> Let's wait few moment till the Impl. Task PED get created.

Now I write jsc#PED-5577 in commit message

File: crmsh/ra.py (hunk 1 above)
Comment (liangxin1300) [reply crmsh/ra.py:54]:
```
# crm_resource --list-standards
ocf
lsb
service
systemd
stonith
```
Just synced with the output of --list-standards

**Review by nicholasyang2022 — APPROVED** [review]

**Pull request merged**